README.md

```diff
@@ -1,2 +1,37 @@
-# mobile-ui-sdks
-Mobile SDKs for authentication and building customizable login UIs for Asgardeo &amp; WSO2 Identity Server.
+<h1 align="center" style="color: #343a40;margin: 20px 0;">
+  <p align="center" style="font-weight: 200;">The <span style="color: #47EBD8">Mobile SDKs</span> for <a href="https://wso2.com">WSO2</a>'s identity and access management products,  <a href="https://wso2.com">WSO2 Identity Server</a> and <a href="https://wso2.com">Asgardeo</a></p>
```

Comment:
Is mentioning both wso2 is and Asgardeo intentional? In all our SDKs, we focus only on Asgardeo.

Reply:
We can go with Asgardeo only

WDYT @brionmario ?

Reply:
Currently going with Asgardeo only, as this repo is in Asgardeo organization